infra/import_graph_test.py

```diff
@@ -0,0 +1,122 @@
+import unittest
+import json
+import os
+from collections import Counter
+
+import import_graph
+
+class TestTypeMatching(unittest.TestCase):
+    def test_node_type_matching(self):
+        # Testing the basic functionality. Cases don't need to be exhaustive;
+        # missing types will be caught during the whole-graph import tests.
+        example_issue = ["sourcecred", "github", "ISSUE", "sourcecred", "sourcecred", "34"]
+        example_repo = ["sourcecred", "github", "REPO", "sourcecred", "sourcecred"]
+        self.assertEqual(import_graph.node_type(example_issue), "github/issue")
+        self.assertEqual(import_graph.node_type(example_repo), "github/repo")
+        with (self.assertRaises(ValueError)):
+            import_graph.node_type(["non","existent", "node"])
+
+    def test_edge_type_matching(self):
+        # Testing the basic functionality. Cases don't need to be exhaustive;
+        # missing types will be caught during the whole-graph import tests.
+        example_has_parent= [
```

Comment:
Suggest using [Black] so that you don’t have to listen to me say “nit:
space before equals sign” all day long. :-)

[Black]: https://github.com/ambv/black

Reply:
I actually had run Black! But then made a few more changes and forgot to re-run it.
Soon(tm): Black will be added to CI for this repository.